db/migrate/20210924193054_add_street_to_fosters_and_add_apt_to_fosters.rb

```diff
@@ -0,0 +1,6 @@
+class AddStreetToFostersAndAddAptToFosters < ActiveRecord::Migration[7.0]
+  def change
+    add_column :fosters, :street, :string
```

Comment:
I would add these new columns to the original migration file that creates the table for simplicity for now, since we don't have data in a live system at the moment.